Gemfile

```diff
@@ -2,19 +2,15 @@ source 'https://rubygems.org'
 
 ruby File.read('.ruby-version').chomp
 
-gem 'actionpack', '~> 7'
-gem 'actionview', '~> 7'
-gem 'activemodel', '~> 7'
-gem 'activerecord', '~> 7'
-gem 'activesupport', '~> 7'
+gem 'rails', '~> 7.1'
```

Comment:
This will pull in actiontext + activestorage + activerecord - I don't think that's necessary for an api only app

Can you swap back to the old lines but update to 7.1

Reply:
I had some trouble with that, some dependencies had conflicts.
I'll try it again, sometimes changing the order fix the them.